example/README.md

```diff
@@ -0,0 +1,48 @@
+## WARNING: This is Alpha software and not intended for use until a stable release.
+
+# Running M3DB on GCP
+
+Setup GCP for m3db:
```

Comment:
Would be useful to draw a picture of the different instances and services they'll end up with once they're done running this guide. I suggest https://monodraw.helftone.com/

Reply:
Cool- I'll work on something

Reply:
Hm, so I was very keen to not advise users to use the static topology.  Any reason why we're adding this?